src/targets/maven.ts

```diff
@@ -490,14 +491,16 @@ export class MavenTarget extends BaseTarget {
       );
     }
 
+    this.logger.info(`Nexus repository closed correctly.`);
     return true;
   }
 
   private getNexusRequestHeaders(): Record<string, string> {
     return {
+      'Content-Type': 'application/json',
```

Comment:
Nexus API is using `Accept` is for get requests, `Content-Type` for post requests, and it needs both.

Reply:
I'd add this as a comment.